mon-pix/tests/acceptance/start-campaigns-test.js

```diff
@@ -88,8 +106,6 @@ describe('Acceptance | Campaigns | Start Campaigns', function() {
               // given
               await visitWithAbortedTransition(`/campagnes/${campaignCode}`);
 
-              expect(currentURL()).to.equal('/campagnes/AZERTY4/identification');
-
```

Comment:
pourquoi enlever cet `expect` ?

Reply:
Il était dans la partie given, et ce use case sera testé dans un it dédié. Au lieu d'avoir de expect un dans le given et un dans le then.